src/tcgwars/logic/impl/gen3/HolonPhantoms.groovy

```diff
@@ -2173,15 +2242,17 @@ public enum HolonPhantoms implements LogicCardInfo {
         text "You can play only one Supporter card each turn. When you play this card, put it next to your Active Pokémon. When your turn ends, discard this card." +
           "Discard a card from your hand. If you can't discard a card from your hand, you can't play this card. Draw 3 cards. If you discarded a Pokémon that has δ on its card, draw 4 cards instead."
         onPlay {
-        my.hand.getExcludedList(thisCard).select("Choose a card to discard.").discard().each {
+          my.hand.getExcludedList(thisCard).select("Discard a card from your hand in order to play ${thisCard}.").discard().each {
             draw 3
             if(it.cardTypes.is(POKEMON) && it.cardTypes.is(DELTA)){
               draw 1
             }
           }
```

Comment:
Since you're replacing a bunch of `findAll`s with `any`s, might as well do this similarly "more correct"
```suggestion
          def toDiscard = my.hand.getExcludedList(thisCard).select("Discard a card from your hand in order to play ${thisCard}.").discard().first
          draw 3
          if(toDiscard.cardTypes.is(POKEMON) && toDiscard.cardTypes.is(DELTA))
            draw 1
```